app/views/pictures/index.html.haml

```diff
@@ -0,0 +1,3 @@
+No pictures yet!
+
+%a(href='#') Add a picture
```

Comment:
Files should end with a trailing newline<br>Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))